tensorflow/tools/pip_package/BUILD

```diff
@@ -128,6 +128,6 @@ sh_binary(
             "//tensorflow/python/saved_model:all_files",
             "//tensorflow/python/tools:all_files",
             "//tensorflow/tensorboard",
-        ],
-    }),
+        ] 
+				}) + if_mkl(["//third_party/mkl:intel_binary_blob"])
```

Comment:
there's a lot of weird whitespacing here that's going to fail our tests, can you try your best to fix these issues, or use buildifier to fix these whitespace problems?